.github/workflows/notified-of-spack-packages-update.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           python-version: '3.10'
       - name: Generate model matrix
         id: generate
-        run: python ./util/matrix_generator.py ${{ github.event.inputs.packages }}
+        run: python ./util/matrix_generation.py ${{ github.event.inputs.packages }}
```

Comment:
Ha... we should have picked that up yesterday!

Reply:
:D I should have picked it up initially too!